common/src/sql/dbwipe.sql

```diff
@@ -8,5 +8,15 @@
  * NOTE: the database and user names MUST be kept in sync with the
  * initialization code and dbwipe.sql.
  */
+
+/*
+ * This is silly, but we throw an error if the user was already deleted.
+ * Create the user so we can always delete it.
+ */
+CREATE DATABASE IF NOT EXISTS omicron;
+CREATE USER IF NOT EXISTS omicron;
+
+ALTER DEFAULT PRIVILEGES FOR ROLE root REVOKE ALL ON TABLES FROM omicron;
```

Comment:
If this is run by the same user that ran dbinit, I think the `FOR ROLE root` is unnecessary.  I mention this because it will be wrong if dbinit _wasn't_ run under that role.  I think we could also use `FOR ALL ROLES`.